README.md

```diff
@@ -86,6 +86,22 @@ WebAssembly backend.
 
 [wasm-pack]: https://rustwasm.github.io/wasm-pack/
 
+### Nix flake
+
+There is a Nix flake available in the `contrib` directory. With, you can get a
+reproducible build of Jotdown by cloning the repo and running
+
+```
+$ nix build ./contrib
```

Comment:
I guess this is for development where it might be good to use 1.56 instead of nightly?

Reply:
I did not setup a derivation for building a development shell, just a derivation for building Jotdown itself. In it, I use the latest Rust, just to get the newest features and speed improvements. If you want, I could setup a development shell managed by Nix as well?